src/test/resources/application-test.yml

```diff
@@ -29,3 +29,6 @@ spring:
 
 
 hmpps.auth.url: http://localhost:8090/auth
+prisoner.search.url: http://localhost:8091/
```

Comment:
minor point. Does this need the / ? the other visit-scheduler.url doesn't have it. Approving anyway!

Reply:
good point. Will check and remove if not needed.

Reply:
Updated